diff --git a/setup.py b/setup.py
index 2a6d12277..68feca4b2 100644
--- a/setup.py
+++ b/setup.py
@@ -375,7 +375,7 @@ build_requires = ['numpy >={},<{}'.format(min_numpy_build_version,
 install_requires = [
     'llvmlite >={},<{}'.format(min_llvmlite_version, max_llvmlite_version),
     'numpy >={},<{}'.format(min_numpy_run_version, max_numpy_run_version),
-    'setuptools<60',
+    'setuptools',
     'importlib_metadata; python_version < "3.9"',
 ]
 

